tests/async/t15148.nim

```diff
@@ -0,0 +1,14 @@
+import asyncdispatch, asyncfile, os
+
+const filename = "t15148.txt"
+
+proc saveEmpty() {.async.} =
+  let
+    text = ""
+    file = open_async(filename, fmWrite)
+  await file.write(text)
+  file.close()
+
+waitFor saveEmpty()
+
+doAssert fileExists(filename)
```

Comment:
```suggestion
```

Not needed. The bug concerns about whether `write()` with empty string works, not whether `openAsync` creates a file.